packages/linux-kernel/arch/arm64/configs/vyos_defconfig

```diff
@@ -2159,21 +2202,119 @@ CONFIG_ISCSI_BOOT_SYSFS=m
 # CONFIG_SCSI_DEBUG is not set
 # CONFIG_SCSI_PMCRAID is not set
 # CONFIG_SCSI_PM8001 is not set
+# CONFIG_SCSI_VIRTIO is not set
```

Comment:
This should ideally be set to =m if we want para-virtualized boot-volumes to work.

Reply:
Good catch, I only tested with normal virtio-blk.
I think we should refresh to 6.x anyway.
On our board (Ten64) 6.2 (will) fixes the SFP issue so I'm very keen to move to that.

Reply:
Im currently running "current" with 6.1.8. Works like a charm with some minor fixes.
Mostly thanks to your ground work ofc :partying_face: